lua/neotest-vstest/init.lua

```diff
@@ -130,6 +130,34 @@ local function create_adapter(config)
     return true
   end
 
+  ---@param fullpath the path of the file to start searching from
+  ---@param stop_at the parent directory to stop searching at
```

Comment:
```suggestion
  ---@param fullpath string the path of the file to start searching from
  ---@param stop_at string the parent directory to stop searching at
```

(https://luals.github.io/wiki/annotations/#param)